examples/esbuild/WORKSPACE

```diff
@@ -25,12 +25,12 @@ http_archive(
     urls = ["https://github.com/bazelbuild/rules_nodejs/releases/download/3.2.2/rules_nodejs-3.2.2.tar.gz"],
 )
 
-_ESBUILD_VERSION = "0.8.48"
+_ESBUILD_VERSION = "0.9.6"
```

Comment:
As there are breaking changes in this version, I don't think we should bump to it. This diff is about adding the update script (change one thing at a time :) )